public/index.html

```diff
@@ -1,71 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-	<link rel="stylesheet" href="/home/victoria/myCVInHtml/public/style.css">
+	<!-- <link rel="stylesheet" href="/home/victoria/myCVInHtml/public/style.css"> -->
+	<link rel="stylesheet" href="/home/victoria/myCVInHtml/public/bootstrap.css">
```

Comment:
link tag should be inside of **head**

Reply:
**bootstrap.css** or **./bootstrap.css** instead of "/home/victoria/myCVInHtml/public/bootstrap.css"